setup.cfg

```
@@ -7,7 +7,7 @@ select =
     RST # flake8-rst-docstrings
 ignore =
     # line breaks should occur before the binary operator
-    W503,
+    W503, W504,
```

```suggestion
    W503,
```
Code that triggers W504 needs to be fixed. We cannot allow line breaks both before and after the operator.